meta/meta.go

```diff
@@ -207,6 +212,26 @@ func (m *Meta) GetAutoRandomID(dbID int64, tableID int64) (int64, error) {
 	return m.txn.HGetInt64(m.dbKey(dbID), m.autoRandomTableIDKey(tableID))
 }
 
+// GenSequenceID adds step to the sequence value and returns the sum.
+func (m *Meta) GenSequenceID(dbID, sequenceID, step int64) (int64, error) {
```

Comment:
```suggestion
func (m *Meta) GenSequenceValue(dbID, sequenceID, step int64) (int64, error) {
```
How about renaming it to `GenSequenceValue`? The name is a little confusing: we pass a parameter `sequenceID` but the function name is `GenSequenceID`.

I know that we're facing a similar problem with the existing `GenAutoTableID`, but I still think renaming the function here is better.

Reply:
addressed